frontend/packages/ux-editor/src/utils/component.test.ts

```diff
@@ -215,7 +215,7 @@ describe('Component utils', () => {
           },
         },
         testProperty4: {
-          $ref: 'https://altinncdn.no/schemas/json/layout/expression.schema.v1.json#/definitions/boolean',
+          $ref: 'expression.schema.v1.json#/definitions/boolean',
```

Comment:
Might be an idea to add this to a constant somewhere since it is used across all the schemas and in both edited files here?